source/common.h

```diff
@@ -60,7 +60,7 @@ std::wstring string2wstring(const std::string& utf8string);
 std::string wstring2string(const std::wstring& widestring);
 
 // Gets position values from ClipBoard
-bool posFromClipboard(int& x, int& y, int& z);
+bool posFromClipboard(Position& position, const int& mapWidth, const int& mapHeight);
```

Comment:
Like in PositionCtrl constructor, you can use int mapWidth= MAP_MAX_WIDTH, int mapHeight= MAP_MAX_HEIGHT to make it optional

Reply:
Should be addressed with this commit https://github.com/hjnilsson/rme/pull/268/commits/b98c4f4c7e05ece1b177249275d8b4d7ce6690dc